src/migrations/1588266891631_smart_contracts.ts

```diff
@@ -61,4 +61,9 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
   pgm.createIndex('smart_contracts', 'microblock_canonical');
   pgm.createIndex('smart_contracts', 'canonical');
   pgm.createIndex('smart_contracts', 'contract_id');
+  pgm.sql('CREATE INDEX index_contract_abi ON smart_contracts USING GIN (abi)');
```

Comment:
Does `pgm.createIndex` have options to configure the index type to be `gin`?

Reply:
No, I spent some time reading the documentation but had no luck.

Reply:
Looking at the ts defs, it seems like this should work:
```ts
pgm.createIndex('smart_contracts', 'abi', { method: 'gin' });
```